cmd/spiderpool-controller/cmd/daemon.go

```diff
@@ -231,6 +232,19 @@ func initControllerServiceManagers(ctx context.Context) {
 	}
 	controllerContext.StsManager = statefulSetManager
 
+	logger.Info("Begin to initialize Subnet Manager")
```

Comment:
if  EnableSpiderSubnet {
     ...... init  
}else{
   log("subnet is disabled")
}